config/configload/merge.go

```diff
@@ -592,6 +621,34 @@ func absInBackends(block *hclsyntax.Block) error {
 	return nil
 }
 
+func addProxy(block *hclsyntax.Block, proxies map[string]*hclsyntax.Block) error {
+	if attr, ok := block.Body.Attributes[proxy]; ok {
+		reference, err := attrStringValue(attr)
+		if err != nil {
+			return err
+		}
+
+		delete(block.Body.Attributes, proxy)
```

Comment:
No need to delete this before throwing error.

Reply:
Done